geomesa-utils/src/main/scala/org/locationtech/geomesa/geomesa.scala

```diff
@@ -1,25 +0,0 @@
-/***********************************************************************
- * Copyright (c) 2013-2017 Commonwealth Computer Research, Inc.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Apache License, Version 2.0
- * which accompanies this distribution and is available at
- * http://www.opensource.org/licenses/apache2.0.php.
- ***********************************************************************/
-
-package org.locationtech
-
-package object geomesa {
-
-  // 0  -> old single table style
-  // 1  -> multi-table style
-  // 2  -> sorted keys in the STIDX table
-  // 3  -> skipped for integration
-  // 4  -> kryo encoded index values
-  // 5  -> z3 index (1.1.0-rc.1 through 1.1.0-rc.2)
-  // 6  -> attribute indices with dates (1.1.0-rc.3 through 1.2.0)
-  // 7  -> z3 polygons (1.2.1)
-  // 8  -> z2 index, deprecating stidx (1.2.2 through 1.2.4)
-  // 9  -> per-attribute-visibilities, not serializing ID in value (not released separately)
-  // 10 -> XZ2, XZ3 (1.2.5+)
-  val CURRENT_SCHEMA_VERSION = 10
```

Comment:
I wonder if anyone is using this variable to figure out which version of GeoMesa's schema they are working with...

Reply:
As mentioned in my other comment, this stopped being using around 1.3.0-ish (maybe earlier given it was last updated in 1.2.5). I am fairly certain no one would be using this, as it's an internal variable - but if they are, then they definitely shouldn't be any more ;)